### DISCUSSION TITLE
Longhorn Persistent Storage on Strimzi for Kafka Cluster

### DISCUSSION THREAD

**Discussion by mit-novian** [strimzi/strimzi-kafka-operator#6098] [Q&A]
Hi, team

I'm trying to use a persistent volume for a kafka cluster. The persistent volume that I use is longhorn, the status of the persistence volume claim is Bound. But the zookeeper's pod from the kafka cluster cannot attach a volume to a persistence volume claim whose the status is already Bound. Here is the yaml i use and the output of the error. Can anyone provide a solution for this?

Thanks

kafka-cluster.yaml
apiVersion: kafka.strimzi.io/v1beta2
kind: Kafka
metadata:
  name: kafka-cluster
spec:
  kafka:
    version: 3.0.0
    replicas: 1
    listeners:
      - name: plain
        port: 9092
        type: internal
        tls: false
      - name: tls
        port: 9093
        type: internal
        tls: true
    config:
      offsets.topic.replication.factor: 1
      transaction.state.log.replication.factor: 1
      transaction.state.log.min.isr: 1
      log.message.format.version: "3.0"
      inter.broker.protocol.version: "3.0"
    storage:
      type: persistent-claim
      size: 1Gi
      class: longhorn
  zookeeper:
    replicas: 1
    storage:
      class: longhorn
      type: persistent-claim
      size: 1Gi
  entityOperator:
    topicOperator: {}
    userOperator: {}
    
    
    
![image](https://user-images.githubusercontent.com/95209163/147233769-857688d4-d5d6-4244-9610-ccffcb2c0fe4.png)

**Comment by scholzj** [comment]
Well, it says that the volume is not mounted. Strimzi doesn't mount the volumes -> that is the responsibility of Kubernetes and the Storage provider. So I think you need to look there why is it not mounted on the node. It could be some issue with the storage I guess, it could be also some scheduling issue where the disk cannot be mounted on every host etc.

As a sidenote - 1Gi of storage is very little for Kafka and might not allow you to do much.
